draft-kuepau-iab-privacy-partitioning.md

```diff
@@ -35,70 +35,83 @@ to analyze such models.
 
 # Introduction
 
-The focus of many common security protocols, such as TLS or IPsec, is to prevent data
-from being modified or seen by parties other than the protocol participants. Encrypting
-and authenticating communication (in HTTP, in DNS, and more) is therefore a prerequisite for
-user privacy by ensuring that information about user identity and activity cannot be
-read by passively observing traffic.
+Protocols such as TLS and IPsec provide a secure (authenticated and encrypted) channel
+between two endpoints over which endpoints transfer information. Encryption and authentication
+of data in transit is necessary to protect information from being seen or modified by parties
+other than the intended protocol participants. As such, it is necessary for ensuring that
```

Comment:
```suggestion
other than the intended protocol participants. As such, this kind of security is necessary for ensuring that
```

Saying "it" didn't make sense, since we are referring to "encryption and authentication", I think